src/main.rs

```diff
@@ -1,56 +1,34 @@
 // Note: Each of the utility files must be added here as a
 // module so that it can be properly included in others.
 mod aggregate_score;
+mod args;
 mod card;
 mod deck;
 mod hand;
 mod hand_score;
 mod hand_stats;
+mod output;
 
-use poker_stats::aggregate_score::sample_aggregate_scores;
-use crate::card::Card;
-use crate::hand::Hand;
-use crate::hand_score::HandScore;
-use crate::hand_stats::HandStats;
-
+use crate::args::{ PokerArgs, StatisticsSampleParameters, CommandsEnum };
+use clap::Parser;
+use output::{draw_hand, print_sorted_deck, getstatistics};
 /// Simple demo for the `poker-stats` crate. For now it
 /// does not support any arguments. It will do three things:
 /// (1) print out the cards in a sorted deck
 /// For each of two randomly drawn hands:
 /// (2) print out the hand
 /// (3) compute and print the card stats
 fn main() {
-    let mut rng = rand::thread_rng();
```

Comment:
Let's keep this declaration at the top-level and pass it down into the individual functions. There are two reasons for this:

(1) It means that if we every want to ensure deterministic output, then we can set the seed from a single point, rather than having to hunt through code 

(2) It is analogous to the "avoid global variables and singletons" idea: making it clear exactly what things each function uses, rather than it grabbing them behind the scenes. This should make the code more portable.

Neither are a big deal here, but good (IMO) to do it anyway.